src/bionics.cpp

```diff
@@ -1920,7 +1920,7 @@ bool player::install_bionics(const itype &type, int skill_level)
         return false;
     }
 
-    // this code should be removed after complete implementation of bionics slots system
+    // @todo: this code should be removed after complete implementation of bionics slots system
```

Comment:
it might be worth checking `doxygen` can still extract this with attached semicolon